.github/actions/set-status/action.yml

```
@@ -0,0 +1,48 @@
+name: 'Set Status'
+description: 'Set status on a specified head-sha'
+inputs:
+  context:
+    description: 'String label of the status'
```

What about just
```suggestion
    description: 'Name of the status'
```
?